spec/lib/TemplateSrvStub.ts

```diff
@@ -4,6 +4,7 @@ export default class TemplateSrvStub {
   variables = [];
   templateSettings = { interpolate : /\[\[([\s\S]+?)\]\]/g };
   data = {};
+  regex = /\$(\w+)|\[\[([\s\S]+?)(?::(\w+))?\]\]|\${(\w+)(?:\.([^:^\}]+))?(?::(\w+))?}/g;
```

Comment:
Is it your regex? Can we have link to eg. regex101 with some to match and not to match examples so it's easy to modify the pattern.

Reply:
No, it is grafana 6.4.4 regexp

Reply:
Can you link to their source (concrete ref) from within the code here to improve future maintainability?

Reply:
done

Reply:
The regex is defined twice here

https://github.com/simPod/grafana-json-datasource/pull/62/files#diff-1ad92e81d89714c517ddefd2ecd50d3dR7
https://github.com/simPod/grafana-json-datasource/pull/62/files#diff-1ad92e81d89714c517ddefd2ecd50d3dR12

Reply:
sorry, wrong conflict resolve